akka-docs/src/main/paradox/java/io-tcp.md

```diff
@@ -1,326 +0,0 @@
-# Using TCP
-
-The code snippets through-out this section assume the following imports:
-
-@@snip [IODocTest.java]($code$/java/jdocs/io/japi/IODocTest.java) { #imports }
-
-All of the Akka I/O APIs are accessed through manager objects. When using an I/O API, the first step is to acquire a
-reference to the appropriate manager. The code below shows how to acquire a reference to the `Tcp` manager.
-
-@@snip [EchoManager.java]($code$/java/jdocs/io/japi/EchoManager.java) { #manager }
```

Comment:
Yes github appears to be confused showing diffs that contain symlinks sometimes... viewing the individual commits sometimes helps.